index.js

```diff
@@ -8,6 +8,10 @@ module.exports = function() {};
 module.exports.pitch = function(remainingRequest) {
     this.cacheable && this.cacheable();
     var query = loaderUtils.parseQuery(this.query);
+    var chunkName = loaderUtils.interpolateName(this, 'routes/' + query.name, {
```

Comment:
For backwards compatibility, should we check the existence of a `[` in the name?  Using the example from the readme, if an existing user had `?name=user`, their behavior would now change.  But maybe that doesn't matter.  Thoughts?